src/servers/src/mysql/handler.rs

```diff
@@ -184,7 +184,7 @@ impl<W: AsyncWrite + Send + Sync + Unpin> AsyncMysqlShim<W> for MysqlInstanceShi
                     user_info = Some(userinfo);
                 }
                 Err(e) => {
-                    increment_counter!(crate::metrics::METRIC_AUTH_FAILURE);
+                    METRIC_AUTH_FAILURE.with_label_values(&[""]).inc();
```

Comment:
What does the label mean? Should we provide a value for it?

Reply:
There was something wrong with the original code, so I changed it accordingly. There should also be a label here.